setup.cfg

```diff
@@ -20,4 +20,4 @@ universal = 1
 upload-dir = docs/_build/html
 
 [pydocstyle]
-add_ignore = D401
+add_ignore = D103,D401
```

Comment:
docstrings are good :-)

Reply:
They are great, I put it there because there were none in `test_books.py`. I 'll take the other route and add them all instead of disabling it.

Reply:
hmm.. I thought you disabled them because of `books/utils.py` (the new one that you add it) That one is more important, the tests are more self explanatory